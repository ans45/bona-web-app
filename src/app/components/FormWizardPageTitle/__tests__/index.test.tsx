import React from 'react';
import { render } from '@testing-library/react';
import { FormWizardPageTitle } from '../';

describe('<FormWizardPageTitle  />', () => {
  it('should match snapshot', () => {
    const { container, getByText } = render(
      <FormWizardPageTitle>Hello World!</FormWizardPageTitle>,
    );

    expect(getByText('Hello World!')).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
