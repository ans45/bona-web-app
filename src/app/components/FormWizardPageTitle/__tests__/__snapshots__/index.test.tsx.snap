// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormWizardPageTitle  /> should match snapshot 1`] = `
<div
  class="MuiBox-root MuiBox-root-3"
>
  <h2
    class="MuiTypography-root makeStyles-root-1 makeStyles-root-2 MuiTypography-h5"
  >
    Hello World!
  </h2>
</div>
`;
